astropy/wcs/wcs.py

```diff
@@ -3014,8 +3014,9 @@ def slice(self, view, numpy_order=True):
                     # equivalently (keep this comment so you can compare eqns):
                     # wcs_new.wcs.crpix[wcs_index] =
                     # (crpix - iview.start)*iview.step + 0.5 - iview.step/2.
-                    crp = ((crpix - iview.start - 1.)/iview.step
-                           + 0.5 + 1./iview.step/2.)
+                    # crp = ((crpix - iview.start - 1.)/iview.step
+                    #        + 0.5 + 1./iview.step/2.)
```

Comment:
Thanks! No point keeping dead code around.
```suggestion
```

Reply:
Just above it, even an older comment was kept. Hence I thought it might have some value.

Reply:
see:
>equivalently (keep this comment so you can compare eqns):

Reply:
@pllim Removed the entire large comment above in the last commit.